posthog/async_migrations/definition.py

```diff
@@ -83,7 +83,11 @@ class AsyncMigrationDefinition:
     def is_required(self) -> bool:
         return True
 
-    # run before the migration and at regular intervals
+    # run before starting the migration
+    def precheck(self) -> Tuple[bool, Optional[str]]:
+        return (True, None)
+
+    # run at a regular interval while the migration is being executed
```

Comment:
do we run the health check in the beginning too? we should, e.g. maybe this ratio is good, but we don't have 100MB left from the beginning